tests/models/swin2sr/test_modeling_swin2sr.py

```diff
@@ -332,3 +332,21 @@ def test_inference_image_super_resolution_head(self):
             [[0.5458, 0.5546, 0.5638], [0.5526, 0.5565, 0.5651], [0.5396, 0.5426, 0.5621]]
         ).to(torch_device)
         self.assertTrue(torch.allclose(outputs.reconstruction[0, 0, :3, :3], expected_slice, atol=1e-4))
+
+    def test_inference_interpolate_pos_encoding(self):
+        processor = Swin2SRImageProcessor.from_pretrained("caidas/swin2SR-classical-sr-x2-64")
+        model = Swin2SRForImageSuperResolution.from_pretrained("caidas/swin2SR-classical-sr-x2-64").to(torch_device)
+
+        image = Image.open("./tests/fixtures/tests_samples/COCO/000000039769.png")
+        image = image.resize([680, 680])  # size is an unrecognized kwargs in processor
```

Comment:
`Swin2SRImageProcessor` doesn’t have size as kwargs, tbh it is not really needed since the model can take any input resolution. 
However, there might be a need to change input size for some reason like having limited hardware resource and want to train on lower resolution images, also to keep consistent code with other image processors. 
If that's the case I'm willing to add it as a PR if approved.